src/languageclient/reloadSchemas.ts

```diff
@@ -0,0 +1,19 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ext } from '../extensionVariables';
+import { ReloadShemasRequest } from './ReloadSchemasRequest';
+
+export async function reloadSchemas(): Promise<void> {
+
+    ext.outputChannel.appendLine("Reloading schemas...");
+
+    if (ext.languageServerClient) {
+        const request = new ReloadShemasRequest();
+        await ext.languageServerClient.sendRequest<ReloadShemasRequest>('schema/reloadSchemas', request);
+    }
+
+    ext.outputChannel.appendLine("Schemas reloaded");
```

Comment:
The schemas won't actually be reloaded at this point (you simply sent a notification to the lang server, right? so await will just wait until the notification is sent, maybe not even received).

How about getting rid of the "Reloading schemas..." and putting "Request to reload schemas sent" or something?

Reply:
I'm not sure how this works in TypeScript/JS - in C# the code would not resume until the request is completed. It would continue if you didn't await and it did a task discard. If it works differently in JavaScript/TS, I'll take your word for it.

Reply:
Could you take a look at this? 
https://stackoverflow.com/questions/43302584/why-doesnt-the-code-after-await-run-right-away-isnt-it-supposed-to-be-non-blo

Reply:
Oh, I see, I'm not sure if await here will actually wait for the response or not. Let me see if I can find documentation on that.

Reply:
Not about TS, it's about your implementation of ISchemaReloadHandler in the lang server as IJsonRpcNotificationHandler<SchemaReloadParams>.  I mentioned in the code review I wasn't sure that would wait for the handler to finish.  Looking at it again, I think it might.  You should just try and see if it waits or not...

Reply:
I double-checked, and as I thought, awaiting on the promise from the sendRequest won't execute any code that come after that. You can try setting a breakpoint on the server or adding await Task.Delay in the server code to verify the client code won't continue executing after await until the server request actually returns. I'm not attached to having two messages, let me know if you still would prefer to have just one - totally fine with that.

Reply:
Then in that case it's only a question of...  A bit pickier...  At that point will the schemas actually have been downloaded and reloaded, or just the cache cleared and the downloads will happen afterwards?  If the former, we're good.  If the latter, might consider a change in second message.  Thx.

Reply:
Good point, schemas are just deleted at that point and the download isn't actually completed. I think I'll just go with a single message for simplicity sake.